src/test/java/org/broadinstitute/hellbender/tools/spark/sv/integration/StructuralVariationDiscoveryPipelineSparkIntegrationTest.java

```diff
@@ -197,9 +198,9 @@ static void svDiscoveryVCFEquivalenceTest(final String generatedVCFPath, final S
             final String experimentalInsDelVcf = onHDFS ? path.toUri().toString() : path.toString();
             actualVcs = extractActualVCs(experimentalInsDelVcf, onHDFS);
 
-            // TODO: 11/30/17 temporary solution to ignore these attributes before they can be brought back
+            // TODO: 1/28/18 see ticket #4228
```

Comment:
Is #4228 referring to SPLIT_READS and READ_PAIRS support annotations? Can you clarify on that ticket which annotations you're talking about? Why don't you have to ignore those attributes here?

Reply:
updated #4228